contrib/plugins/qt-creator-project.py

```diff
@@ -0,0 +1,229 @@
+    outFile.write(' <value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">Build</value>\n')
+    outFile.write(' <value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName"></value>\n')
+    outFile.write(' <value type="QString" key="ProjectExplorer.ProjectConfiguration.Id">ProjectExplorer.BuildSteps.Build</value>\n')
+    outFile.write('</valuemap>\n')
+    outFile.write('<value type="int" key="ProjectExplorer.BuildConfiguration.BuildStepListCount">1</value>\n')
+    outFile.write('<value type="bool" key="ProjectExplorer.BuildConfiguration.ClearSystemEnvironment">false</value>\n')
+    outFile.write('<valuelist type="QVariantList" key="ProjectExplorer.BuildConfiguration.UserEnvironmentChanges">\n')
+    outFile.write(' <value type="QString">' + "PATH=" + os.environ['PATH'] + '  </value>\n')
+    outFile.write('</valuelist>\n')
+    outFile.write('<value type="QString" key="ProjectExplorer.ProjectConfiguration.DefaultDisplayName">Vorgabe</value>\n')
+    outFile.write('<value type="QString" key="ProjectExplorer.ProjectConfiguration.DisplayName">' + name + '</value>\n')
+    outFile.write('<value type="QString" key="ProjectExplorer.ProjectConfiguration.Id">GenericProjectManager.GenericBuildConfiguration</value>\n')
+    outFile.write('<valuelist type="QVariantList" key="UserStickyKeys">\n')
+    outFile.write(' <value type="QString">ProjectExplorer.BuildConfiguration.UserEnvironmentChanges</value>\n')
+    outFile.write(' <value type="QString">UserStickyKeys</value>\n')
+    outFile.write('</valuelist>\n')
+    outFile.write('</valuemap>\n')
+
+def addBuildSteps(outFile, buildMeFile):
+    addBuildConfig(outFile, 0, "Bob dev ", "" , buildMeFile)
+    addBuildConfig(outFile, 1, "Bob dev (force)", "-f", buildMeFile)
+    addBuildConfig(outFile, 2, "Bob dev (no checkout)", "-b", buildMeFile)
+    addBuildConfig(outFile, 3, "Bob dev (no deps)", "-n", buildMeFile)
+    addBuildConfig(outFile, 4, "Bob dev (no checkout, no deps)", "-nb", buildMeFile)
+
+    outFile.write('<value type="int" key="ProjectExplorer.Target.BuildConfigurationCount">5</value>\n')
+    outFile.write('<valuelist type="QVariantList" key="UserStickyKeys">\n')
+    outFile.write(' <value type="QString">ProjectExplorer.BuildConfiguration.BuildStepListCount</value>\n')
+    outFile.write(' <value type="QString">ProjectExplorer.BuildConfiguration.UserEnvironmentChanges</value>\n')
+    outFile.write(' <value type="QString">ProjectExplorer.ProjectConfiguration.DisplayName</value>\n')
+    outFile.write('</valuelist>\n')
+
+def generateQtProject(package, destination, updateOnly, args):
+    project = "/".join(package.getStack())
+
+    dirs = []
+    getCheckOutDirs(package, dirs)
+    projectDir = project.replace('/','_')
+    if projectDir[-1:] == '_':
+        projectDir =projectDir[:-1]
+
+
+    if not destination:
+        destination = os.path.join(os.getcwd(), projectDir)
```

Comment:
I agree with the first point. Maybe not use "project" but "qt-project" or "project/qt-project". Otherwise we'll probably mix up project files.

For the second: If I use `package.getName()` for `projectDir` I'll lose the possibility to have the same package with different settings open in parallel. This will happen in case you have a app cross-compiled for the target `bob project target/whatever/stuff/my_app` and also for the host `bob project host/whatever/stuff/my_app`. On the other hand to long project name does not disturb me. A good tradeoff is maybe another parameter e.g. `bob project --name=host_myapp`?